Cargo.toml

```diff
@@ -10,12 +10,14 @@ anyhow = "1.0"
 async-trait = "0.1"
 base64 = "0.13"
 bytes = "1"
+fs_extra = "1.2"
```

Comment:
Not the most well-maintained codebase ever but it'll do, I don't think there's a better alternative at the moment.

Reply:
Noted, thanks.